### ISSUE TITLE
[site:new] Does not ask for site-name

### ISSUE THREAD

**Issue by Sutharsan** [hechoendrupal/DrupalConsole#1733]
On version: 0.10.4
When executing ```drupal site:new``` the response is:
```
 Getting releases for Drupal

 Select a core release:
  [0 ] 8.0.2
  [1 ] 8.0.1
  [2 ] ... (and many more)
```
I enter '0'
The the response is: 
```                                         
  [RuntimeException]                            
  Not enough arguments (missing: "site-name").  
                                                

site:new <site-name> [<version>]
```
It would be more user friendly to ask for the site-name first.

**Comment by jmolivas** [comment]
@Sutharsan looking at this and having it fixed. 

Does look more sense to rename the `site-name` argument as `directory'?

**Comment by Sutharsan** [comment]
Yes it does.

**Comment by Sutharsan** [comment]
Although, it was clear what I had to enter there.

**Comment by jmolivas** [comment]
PR https://github.com/hechoendrupal/DrupalConsole/pull/1735 sent juts waiting for travis to pass the build.

**Comment by jmolivas** [comment]
@Sutharsan: This change is included in the latest v0.10.5 release.
